ee/src/main/java/org/jboss/as/ee/subsystem/ManagedExecutorServiceAdd.java

```diff
@@ -105,25 +103,25 @@ protected void performRuntime(OperationContext context, ModelNode operation, Mod
         final AbstractManagedExecutorService.RejectPolicy rejectPolicy = AbstractManagedExecutorService.RejectPolicy.valueOf(ManagedExecutorServiceResourceDefinition.REJECT_POLICY_AD.resolveModelAttribute(context, model).asString());
 
         final ManagedExecutorServiceService service = new ManagedExecutorServiceService(name, jndiName, hungTaskThreshold, longRunningTasks, coreThreads, maxThreads, keepAliveTime, keepAliveTimeUnit, threadLifeTime, queueLength, rejectPolicy);
-        final ServiceBuilder<ManagedExecutorServiceAdapter> serviceBuilder = context.getServiceTarget().addService(ConcurrentServiceNames.getManagedExecutorServiceServiceName(name), service);
-
-        final boolean rcPresent = context.hasOptionalCapability("org.wildfly.request-controller", null, null);
+        final CapabilityServiceBuilder serviceBuilder = context.getCapabilityServiceTarget()
```

Comment:
Unless you are willing to refactor ManagedExecutorServiceService to support MSC 1.4.x, you need to use:
```java
context.getCapabilityServiceTarget().addCapability(ManagedExecutorServiceResourceDefinition.CAPABILITY, service);
```

Reply:
done